.cirrus.yml

```diff
@@ -0,0 +1,44 @@
+task:
+  only_if: $CIRRUS_BRANCH == 'main' || $CIRRUS_BRANCH == 'develop' || $CIRRUS_TAG != '' || $CIRRUS_PR != ''
+  env:
+    CIRRUS_CLONE_SUBMODULES: true
+
+  matrix:
+    - name: Test build
+
+      # Default is 2 CPUs and 4 GB of memory
```

Comment:
These jobs just take a lot of memory

![lots-of-memory](https://user-images.githubusercontent.com/5142394/210636097-9503982d-9b95-48ee-8053-0752a91a6cef.png)